component/FloatingAction.js

```diff
@@ -396,6 +396,18 @@ class FloatingAction extends Component {
     }
     this.reset();
   }
+  
+  _updateState = (nextState, callback) => {
+    const { onStateChange } = this.props;
+    this.setState(nextState, () => {
+      if (typeof callback == 'function') {
```

Comment:
not necessary to check if callback and onStateChange are  functions... just use the following check `if (callback)` and `if (onStateChange)`